docs/Makefile

```diff
@@ -11,7 +11,7 @@ PAPER         =
 PAPEROPT_a4     = -D latex_paper_size=a4
 PAPEROPT_letter = -D latex_paper_size=letter
 #ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
-ALLSPHINXOPTS   = -j 2 source
+ALLSPHINXOPTS   = -j 1 source
```

Comment:
Was this an intentional change? If so, what is the purpose of it?

Reply:
Pretty arbitrary: By default we run the sphinx build with 2 PEs. but sometimes (on my laptop) that's slower than with just one.

Reply:
Fair enough, that's fine by me! Thought I'd check since the change seemed unrelated to the core of the PR...